projects/aws/image-builder/builder/utils.go

```diff
@@ -109,19 +112,70 @@ func execCommand(cmd *exec.Cmd) (string, error) {
 	return commandOutputStr, nil
 }
 
-func getGitCommitFromBundle(repoPath string) (string, error) {
+func (bo *BuildOptions) getGitCommitFromBundle() (string, error) {
 	log.Println("Getting git commit from bundle")
-	loadBundleManifestCommandSequence := fmt.Sprintf("source %s/build/lib/eksa_releases.sh && build::eksa_releases::load_bundle_manifest %s", repoPath, os.Getenv("EKSA_USE_DEV_RELEASE"))
-	cmd := exec.Command("bash", "-c", fmt.Sprintf("%s", loadBundleManifestCommandSequence))
-	commandOut, err := execCommand(cmd)
+	releasesManifestContents, err := readFileFromURL(eksAReleasesManifestURL)
+	if err != nil {
+		return "", err
+	}
+
+	releases := &releasev1.Release{}
+	if err = yaml.Unmarshal(releasesManifestContents, releases); err != nil {
+		return "", fmt.Errorf("failed to unmarshal release manifest from [%s]: %v", eksAReleasesManifestURL, err)
+	}
+
+	var eksAReleaseVersion, bundleManifestUrl string
+	var foundRelease bool
+	if bo.EKSAReleaseVersion != "" {
+		eksAReleaseVersion = bo.EKSAReleaseVersion
+	} else {
+		eksAReleaseVersion = releases.Spec.LatestVersion
```

Comment:
lets put out a warning to the user here to make it clear we are using the latest

`No eks-a-release specified, using the latest version of <>`

Reply:
Done